datadog_checks_dev/datadog_checks/dev/kind.py

```diff
@@ -19,7 +19,7 @@
 
 
 @contextmanager
-def kind_run(sleep=None, endpoints=None, conditions=None, env_vars=None, wrappers=None):
+def kind_run(sleep=None, endpoints=None, conditions=None, env_vars=None, wrappers=None, kind_config=None):
```

Comment:
I made a separate PR for adding support for a kind configuration file: https://github.com/DataDog/integrations-core/pull/9930
Should I still keep this change or should I wait until this other PR is merged first?

Reply:
Merged!